src/Transport/Config/AzureServiceBusTransportInfrastructure.cs

```diff
@@ -27,7 +27,19 @@ public override EndpointInstance BindToLocalEndpoint(EndpointInstance instance)
 
         public override string ToTransportAddress(LogicalAddress logicalAddress)
         {
-            return logicalAddress.ToString();
+            var queue = new StringBuilder(logicalAddress.QueueName);
```

Comment:
Thank you @SzymonPobiega.
So discriminator and qualifier are mutually exclusive? Sorry for pestering with these questions, want to understand :)